src/main/java/uk/gov/justice/hmpps/prison/repository/sql/PrisonerRepositorySql.kt

```diff
@@ -48,7 +48,7 @@ enum class PrisonerRepositorySql(val sql: String) {
 
   LIST_ALL_OFFENDERS(
     """
-        SELECT OFFENDER_ID_DISPLAY AS OFFENDER_NUMBER, max(OFFENDER_ID) AS mo FROM OFFENDERS GROUP BY OFFENDER_ID_DISPLAY ORDER BY mo ASC
+        SELECT OFFENDER_ID_DISPLAY AS OFFENDER_NUMBER, max(ROOT_OFFENDER_ID) AS mo FROM OFFENDERS GROUP BY OFFENDER_ID_DISPLAY ORDER BY mo ASC
     """
```

Comment:
new offenders will always be at the end of the list rather then new offenders and ones where an alias was created on an old offender